src/satellites.jl

```diff
@@ -543,4 +603,23 @@ function rmFile(fileName, printLevel = 1)
             printstyled("(file $fileName does not exist - skipping).\n", color=:red)
         end
     end
+end
+
+# Statically split range [1,N] into equal sized chunks for np processors
+function splitrange(N::Int, np::Int)
```

Comment:
can you add a more extensive documentation here?

Reply:
I would appreciate if you could do this, don't know exactly what you did

Reply:
I didn't write the `splitrange` function. Can you combine it with the `splitting` function?